docs/source/reference/routines.rst

```diff
@@ -30,14 +30,14 @@ These functions cover a subset of
    ext
 
 
-CUB backend for reduction routines
+CUB/cuTENSOR backend for reduction routines
 ----------------------------------
 Some CuPy reduction routines, including :func:`~cupy.sum`, :func:`~cupy.min`, :func:`~cupy.max`,
 :func:`~cupy.argmin`, :func:`~cupy.argmax`, and other functions built on top of them, can be
-accelerated by switching to the `CUB`_ backend. The switch can be toggled on or off at runtime
-by setting the bool :data:`cupy.cuda.cub_enabled`, which is set to ``False`` by default. Note
-that while in general CUB-backed reductions are faster, there could be exceptions depending on
-the data layout. We recommend users to perform some benchmarks to determine whether CUB offers
-better performance or not.
+accelerated by switching to the `CUB`_ or `cuTENSOR`_ backend. These backends can be enabled
+by setting ``CUPY_ACCELERATORS`` environement variable. Note that while in general the accelerated
```

Comment:
Is it possible to add a hyperlink to ``CUPY_ACCELERATORS`` in the environment variable page?